src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -320,16 +321,18 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('application_end_port', options_list=['--application-end-port', '--app-end-port'], help='Application End port of a range of ports.')
         c.argument('ephemeral_start_port', help='Ephemeral start port of a range of ports.')
         c.argument('ephemeral_end_port', help='Ephemeral end port of a range of ports.')
-        c.argument('vm_size', help='The size of virtual machines in the pool. All virtual machines in a pool are the same size.', default='Standard_D2')
+        c.argument('vm_size', help='The size of virtual machines in the pool. All virtual machines in a pool are the same size.')
         c.argument('vm_image_publisher', help='The publisher of the Azure Virtual Machines Marketplace image.', default='MicrosoftWindowsServer')
         c.argument('vm_image_offer', help='The offer type of the Azure Virtual Machines Marketplace image.', default='WindowsServer')
-        c.argument('vm_image_sku', help='The SKU of the Azure Virtual Machines Marketplace image.', default='2019-Datacenter')
+        c.argument('vm_image_sku', help='The SKU of the Azure Virtual Machines Marketplace image.')
         c.argument('vm_image_version', help='The version of the Azure Virtual Machines Marketplace image. ', default='latest')
         c.argument('capacity', arg_type=capacity)
         c.argument('placement_property', arg_type=placement_property)
         c.argument('is_stateless', arg_type=get_three_state_flag(), help='Indicates if the node type can only host Stateless workloads.', default=False)
         c.argument('multiple_placement_groups', options_list=['--multiple-placement-groups', '--multi-place-groups'], arg_type=get_three_state_flag(),
                    help='Indicates if scale set associated with the node type can be composed of multiple placement groups.', default=False)
+        c.argument('security_type', arg_type=get_enum_type(SecurityType), help='Specifies the security type of the nodeType. Only TrustedLaunch is currently supported.')
+        c.argument('secure_boot_enabled', arg_type=get_three_state_flag(), help='Specifies whether secure boot should be enabled on the nodeType. Can only be used with TrustedLaunch SecurityType.')
```

Comment:
Please use the first-person voice in the help message
```suggestion
        c.argument('security_type', arg_type=get_enum_type(SecurityType), help='Specify the security type of the nodeType. Only TrustedLaunch is currently supported.')
        c.argument('secure_boot_enabled', arg_type=get_three_state_flag(), help='Specify whether secure boot should be enabled on the nodeType. Can only be used with TrustedLaunch SecurityType.')
```